src/main/java/com/fasterxml/jackson/core/JsonGenerator.java

```diff
@@ -2977,4 +3008,18 @@ protected void _writeSimpleObject(Object value) throws IOException
         throw new IllegalStateException("No ObjectCodec defined for the generator, can only serialize simple wrapper types (type passed "
                 +value.getClass().getName()+")");
     }
+
+    /**
+     * Returns a new {@code int[]} of escape table with additional forward-slash escaping configured
+     * as configured by {@link JsonWriteFeature#ESCAPE_FORWARD_SLASHES}.
+     *
+     * @since 2.17
+     */
+    protected int[] escapeForwardSlash(int[] escapes) {
+        int[] esc = Arrays.copyOf(escapes, escapes.length);
```

Comment:
I think this should be optimized a bit by checking if any change is actually needed -- and only making defensive copy in that case.

However, this leads to my main concern here: copying of the array for every generator instance adds overhead, which can be non-trivial in case of writing small documents.
So I wonder if there's any way to avoid this overhead by pre-creating escaping table in `CharTypes`.